core/src/main/res/layout/mz_view_map.xml

```diff
@@ -11,6 +11,20 @@
       android:layout_height="match_parent"
       />
 
+  <com.mapzen.android.graphics.CompassView
+      android:id="@+id/mz_compass"
+      android:layout_width="@dimen/mz_compass_button_width"
+      android:layout_height="@dimen/mz_compass_button_height"
+      android:layout_alignParentLeft="true"
```

Comment:
I didn't realize the sdk differed from the app :) Given that, I retract my comment, the compass should be in the top left. I've created a ticket here to provide an alternate configuration for developers who want consistency with the app:
https://github.com/mapzen/android/issues/241